setup.py

```diff
@@ -1,11 +1,15 @@
 import os
+import sys
 
 from setuptools import setup, find_packages
 
 
-version = '1.3.dev0'
+version = '2.0.dev0'
 description = "Importchecker finds unused imports in Python modules."
-test_requirements = ['mock', 'zope.testrunner']
+test_requirements = ['zope.testrunner']
+
+if sys.version_info.major == 2:
+    test_requirements.append('mock')
```

Comment:
This sort of dynamic dependency computation used to cause problems with certain versions of pip due to wheel caching.

The modern alternative is to use environment markers: https://hynek.me/articles/conditional-python-dependencies/